README.md

```diff
@@ -9,4 +9,8 @@ In an effort to keep the examples easy to follow, much of the data processing an
 1. `xcom_gcs_ds.py` - A simple DS pipeline from data extraction to modeling.
     - Pulls data from BigQuery using the Google Provider (BigQueryHook) into a dataframe that preps, trains, and builds the model
     - Data is passed between the tasks using [XComs](https://airflow.apache.org/docs/apache-airflow/stable/concepts/xcoms.html)
-    - Uses GCS as an Xcom backend to easily track intermediary data in a scalable, external system
\ No newline at end of file
+    - Uses GCS as an Xcom backend to easily track intermediary data in a scalable, external system
+
+2. `xcom_gcs_ds_k8sExecutor.py` - A simple DS pipeline from data extraction to modeling that leverages the flexibility of the [Kubernetes Executor](https://www.astronomer.io/blog/new-kubernetesexecutor).
```

Comment:
Might be worth mentioning that this pipeline should really _only_ be used with the `KubernetesExecutor` so others know the particular use case for this example vs. the original one. WDYT?

Reply:
For sure, will update.